docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -345,22 +344,14 @@ For airgapped environments, make sure your Yum repository mirror contains these
 
 ##### Install Prometheus node exporter
 
-For YugabyteDB Anywhere versions 2.8 and later, download the 1.3.1 version of the Prometheus node exporter, as follows:
+Download the 1.3.1 version of the Prometheus node exporter, as follows:
 
 ```sh
 wget https://github.com/prometheus/node_exporter/releases/download/v1.3.1/node_exporter-1.3.1.linux-amd64.tar.gz
 ```
 
-For YugabyteDB Anywhere versions prior to 2.8, download the 0.13.0 version of the exporter, as follows:
-
-```sh
-wget https://github.com/prometheus/node_exporter/releases/download/v0.13.0/node_exporter-0.13.0.linux-amd64.tar.gz
-```
-
 If you are doing an airgapped installation, download the node exporter using a computer connected to the internet and copy it over to the database nodes.
 
-Note that the instructions are for the 0.13.0 version. The same instructions are applicable to the 1.3.1 version, but you need to use the correct file name.
-
 On each node, perform the following as a user with sudo access:
 
 1. Copy the `node_exporter-....tar.gz` package file that you downloaded into the `/tmp` directory on each of the YugabyteDB nodes. Ensure this file is readable by the `centos` user on each node (or another user with sudo privileges).
```

Comment:
Some of the wording around sudo access could probably be consolidated. We should maybe be less specific about the actual username `centos`, since this username may vary from install to install.

The full filename of the tarball could be specified here.

Reply:
Fixed